addon/components/vertical-collection/component.js

```diff
@@ -29,18 +29,54 @@ const VerticalCollection = Component.extend({
 
   tagName: 'vertical-collection',
 
+  /**
+   * Property name used for storing references to each item in items. Accessing this attribute for each item
+   * should yield a unique result for every item in the list.
+   * 
+   * @property key
+   * @type String
+   * @default '@identity'
+   */
   key: '@identity',
 
   // –––––––––––––– Required Settings
 
+  /**
+   * Estimated height of an item to be rendered. Use best guess as this will be used to determine how many items
+   * are displayed virtually, before and after the vertical-collection viewport.
+   *
+   * @property estimateHeight
+   * @type Number
+   * @required
+   */
   estimateHeight: null,
 
-  // usable via {{#vertical-collection <items-array>}}
+  /**
+   * List of objects to svelte-render.
+   * Can be called like `{{#vertical-collection <items-array>}}`, since it's the first positional parameter of this component.
+   *
+   * @property items
+   * @type Array
+   * @required
+   */
   items: null,
 
   // –––––––––––––– Optional Settings
+  /**
+   * Indicates if this component's height will change after initial render.
+   * If true, item tracking will be done using the StaticRadar, otherwise the DynamicRadar.
```

Comment:
@pzuraq confirm that better wording would be `Indicates that this component's height might change after initial render or may differ from 'estimateHeight'`

Reply:
Agree, but the opposite: `Indicates if the occluded item's heights will change or not. If true, the vertical-collection will assume that item height is always equal to estimateHeight. This is more performant, but less flexible.`